game/libda/mixer.hpp

```diff
@@ -8,31 +8,29 @@
  */
```

Comment:
these files don't really follow the naming conventions of the rest of the project, suggesting that they are external dependencies which shouldn't be modified. if they aren't, please rename them.

Reply:
libda is my own project which only lives within the Performous repository and is hardly maintained. The couple of includes to ../something are a bit curious (not by me) as libda is intended not to depend on Performous. However, these issues are beyond the purpose of this branch; but changing away from Boost to stdlib still makes sense and is necessary for libda as well.